osutil/strace/export_test.go

```diff
@@ -20,7 +20,7 @@
 package strace
 
 var (
-	ExcludedSyscalls = excludedSyscalls
+	ExcludedSyscalls = getExcludedSyscalls()
```

Comment:
This appears to be unused

Reply:
It is used in the strace-test package, i.e. the other _test.go files inside strace package.

I.e. this public export, is for strace package tests only, and not usable by tests inside any other packages....

Reply:
Silly me, I was looking at the diff only!